java/api/src/main/java/io/ray/api/runtimeenv/RuntimeEnv.java

```diff
@@ -12,14 +14,24 @@ public interface RuntimeEnv {
   public static class Builder {
 
     private Map<String, String> envVars = new HashMap<>();
+    private List<String> jars = new ArrayList<>();
 
     public Builder addEnvVar(String key, String value) {
```

Comment:
We can also add a brief doc for this function?

Reply:
done